BLSQ/iaso
# WC2-107: auto blacklist old nfc cards

New feature for NFC card management:

Given the backend receives a new “WRITE_PROFILE” event
When the entity associated with that event has other cards associated to them
Then those other cards are automatically blacklisted with : 
 - the reason "OTHER" 
 - and the comment "Profile was written on {NEW NFC CARD ID}"

## Self proof reading checklist


## How to test

Mobile-related endpoint, so difficult to test, two solutions come to mind:

- check the newly added tests in `test_storage.py` make sense and work
- for @bmonjoie: test the interactions directly from the mobile app
